lib/Conversion/HandshakeToHW/HandshakeToHW.cpp

```diff
@@ -307,7 +307,7 @@ static LogicalResult convertExtMemoryOps(HWModuleOp mod) {
   OpBuilder b(mod);
 
   auto getMemoryIOInfo = [&](Location loc, Twine portName, unsigned argIdx,
-                             ArrayRef<hw::PortInfo> info,
+                             ArrayRef<hw::PortInfo *> info,
```

Comment:
Why was it necessary to move to a pointer here?

Reply:
consequence of ArrayRef from a serquential container's iterators.